tests/system/testdata/sessions/events.txtar

```diff
@@ -49,16 +49,18 @@ project:
   triggers:
     - name: http_start
       type: webhook
-      call: program.star:on_http_start
+      call: program.py:on_http_start
     - name: http1
       type: webhook
     - name: http2
       type: webhook
 
--- program.star --
-def on_http_start():
-  s1 = subscribe("http2")
-  s2 = subscribe("http1")
+-- program.py --
+from autokitteh import subscribe, next_event
+
+def on_http_start(_):
+  s1 = subscribe("http2", "true")
+  s2 = subscribe("http1", "true")
```

Comment:
Can comment on lines below, but subscribe works with one argument that can be a `str` or `list[str]`

```python
  print(4, next_event([s1, s2], timeout=2)['body']['bytes'])
  print(5, next_event([s1, s2], timeout=2)['body']['bytes'])
```

Reply:
done